model/search/ResultsWatcher.php

```diff
@@ -114,4 +129,21 @@ public function isResultSearchEnabled()
     {
         return (bool)$this->getOption(self::OPTION_RESULT_SEARCH_FIELD_VISIBILITY);
     }
+
+    private function transformDateTime(string $getStartTime): string
+    {
+        $timeArray = explode(" ", $getStartTime);
+        $date = DateTimeImmutable::createFromFormat('U', $timeArray[1]);
+
+        if ($date === false) {
+            $date = DateTimeImmutable::createFromFormat('Y-m-d H:i:s', $getStartTime);
+        }
+
+        if (!$date instanceof DateTime) {
+            $this->logCritical('We wer not able to transform delivery-execution start time!');
```

Comment:
Can you please add the the `$deliveryExecution->getUri` and the `$getStartTime` to the logs so we can find the record with problem and fix it?